ebib.el

```diff
@@ -4017,8 +4024,7 @@ With a prefix argument PFX, edit the current field directly,
 without any special treatment.  Exceptions are the \"type\" field
 and any field with a multiline value, which are always edited in
 a special manner."
-  (interactive "p")
-  (let* ((field (ebib--current-field))
+  (let* ((pfx (car current-prefix-arg))
```

Comment:
AFAIU `current-prefix-arg` is not always a list, so taking the `car` of it is not safe. The best method is probably to read the prefix argument in `ebib-edit-current-field` and pass it as a second optional argument to `ebib--edit-field`. Or less convoluted, use `current-prefix-arg` directly. We only want to know whether it's `nil` or not, so there's no need to examine it more closely.

Reply:
Sorry, just realised I hadn't dealt with this. You're right, it is a problem. We do check whether it's nil or not, but we also run `numberp` on it a bit earlier in the function. But both could be reduced to checking against nil, so I'll rewrite it that way.

Reply:
> Or less convoluted, use current-prefix-arg directly

Sorted. The implementation I ended up with is almost stupidly simple, but it seems to work.